components/service/fxa/.gitignore

```diff
@@ -0,0 +1,3 @@
+# jniLibs
+*.so
+!libjnidispatch.so
```

Comment:
Can we move this to the root .gitignore file? I think we should get rid of all the project ones.